dropwizard-testing/src/main/java/io/dropwizard/testing/junit/ResourceTestRule.java

```diff
@@ -37,11 +38,14 @@
      */
     public static class Builder {
 
+        private static Consumer<ClientConfig> NOOP_CONSUMER = c -> {};
```

Comment:
I think the lambda expression could be just inlined with the consumer declaration, because we don't reference it in other places.